orchestrator/taste-orchestrator.py

```diff
@@ -1090,6 +1141,11 @@ def InvokeOcarinaMakefiles(
                         base = os.path.basename(ss)
                         baseDir = os.path.splitext(base)[0]
                         externals += g_absOutputDir + os.sep + baseDir + os.sep + "ext" + os.sep + '*.o '
+                for ss in rosBridgeSubsystems:
+                    if ss == aplc:
+                        base = os.path.basename(ss)
+                        baseDir = os.path.splitext(base)[0]
+                        externals += g_absOutputDir + os.sep + baseDir + os.sep + "ext" + os.sep + '*.o '
```

Comment:
Taste wise, I'd prefer `os.sep.join(...)` and consistency in string quotation style.

Reply:
Done